UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -557,7 +557,7 @@
 
   UefiCpuPkg/CpuDxe/CpuDxe.inf
   MdeModulePkg/Universal/BdsDxe/BdsDxe.inf
-  UefiPayloadPkg/Logo/Logo.inf
+  MdeModulePkg/Logo/Logo.inf

```

Comment:
This is wrong. It won't compile your custom Logo driver now @mkopec 
Here and below applies the same comment

Reply:
This .inf only describes the bmp file, i thought that if include MdeModulePkg's version here, the file from there would be used in UefiPayloadPkg. Turns out it doesn't make a difference.

Reverted to `UefiPayloadPkg/Logo/Logo.inf`.

Reply:
In that case both versions should work although it would have to be tested